stern/tail.go

```diff
@@ -131,6 +132,16 @@ func (t *Tail) Start(ctx context.Context, i v1.PodInterface) {
 				}
 			}
 
+			matches := false
```

Comment:
Can you first check if we need to do these checks? 

Something like this (haven't tested it): 
```suggestion
			if len(t.Options.Include) != 0 {
				matches := false
				for _, rin := range t.Options.Include {
					if rin.MatchString(str) {
						matches = true
						break
					}
				}
 				if !matches {
					continue OUTER
				}
			}
```

Also added a break if it does match the input.